crowbar_framework/app/models/crowbar_service.rb

```diff
@@ -62,8 +62,8 @@ def transition(inst, name, state)
         return [404, "Node not found"]
       end
 
-      if state == "readying"
-        transition_to_readying inst, name, state, node
+      if state == "ready" && !node.admin?
```

Comment:
I'm not sure how big change do we want for Cloud5 (but the bug is in current code as well).